apps/chat/src/components/Chat/ModelVersionSelect.tsx

```diff
@@ -37,6 +44,20 @@ export const ModelVersionSelect = ({
   );
 
   if (entities.length < 2) {
+    if (entities.length && entities[0].version) {
+      return (
+        <p className="flex gap-2" data-qa="version">
+          {showVersionPrefix && (
+            <>
+              <span className="hidden md:block">{t('version: ')}</span>
+              <span className="md:hidden">{t('v: ')}</span>
+            </>
```

Comment:
this block is used at least twice. Maybe better to create small internal component and use in both places and export it if somewhere we have the same rendering